trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name[0...10]
+  end
```

Comment:
- [ ] it should only trim the name if the name is longer than 10 character.
- [ ] your code right now trims on 11 characters please change it to `@namable.correct_name[0...9]`